cellarium/cas/_io.py

```diff
@@ -145,6 +145,29 @@ def read_10x_h5(
     return adata
 
 
+def read_h5_or_h5ad(filename: str) -> "anndata.AnnData":
```

Comment:
Can this just be:
```suggestion
def read_h5_or_h5ad(filename: str) -> anndata.AnnData:
```
?